homeassistant/components/vicare/strings.json

```diff
@@ -424,6 +424,35 @@
       },
       "compressor_phase": {
         "name": "Compressor phase"
+      },
+      "ventilation_demand": {
+        "name": "Ventilation demand",
+        "state": {
+          "unknown": "Unknown",
+          "ventilation": "Ventilation"
+        }
+      },
```

Comment:
What does this sensor do?

Reply:
So for my device it can just display these two values, could be more for other devices. Maybe it does not bring that much value, I can also remove it.